aries-site/src/pages/components/controls.js

```diff
@@ -70,29 +78,36 @@ const Controls = () => (
         >
           <ButtonExample />
         </Example>
-        <SubsectionText>
-          Buttons are used to indicate actions that can be performed.
-        </SubsectionText>
       </Subsection>
     </ContentSection>
     <ContentSection>
       <Subsection name="Menu">
-        <MenuExample />
-        <SubsectionText>
+      <SubsectionText>
           Menu is used to filter or sort content on a page. It is similar to the
           select component. However, use the select component when the user must
```

Comment:
Can we add an `<Anchor />` around "select component" and link it to the Select component deep link? Also capitalize Select in both instances as select is the name of the component and thus a proper noun which should be capitalized.

Reply:
good thinking! I added that in